app/containers/Forms/EmissionSourceFields.tsx

```diff
@@ -8,16 +8,16 @@ const EmissionSourceFields: React.FunctionComponent<FieldProps> = ({
   return (
     <Row style={{margin: '20px 0 20px 0'}} className="source-field-heading">
       <Col xs={12} md={4}>
-        <h5 style={{textAlign: 'left'}}>{formData}</h5>
+        <h2 style={{textAlign: 'left'}}>{formData}</h2>
       </Col>
       <Col xs={12} md={3}>
-        <h6>Tonnes</h6>
+        <p className="unit">Tonnes</p>
       </Col>
       <Col xs={12} md={2}>
-        <h6>GWP</h6>
+        <p className="unit">GWP</p>
       </Col>
       <Col xs={12} md={3}>
-        <h6>Tonnes (CO2e)</h6>
+        <p className="unit">Tonnes (CO2e)</p>
```

Comment:
Rule: semantic headings

These `h6`s have been changed to paragraphs because, as unit names, they did not delineate content sections of the page. (ie: sections that you'd navigate directly to through a screenreader menu)